cluster/ephemeral-provider-common.sh

```diff
@@ -29,11 +29,18 @@ function _registry_volume() {
 }
 
 function _add_common_params() {
-    local params="--nodes ${KUBEVIRT_NUM_NODES} --memory 4096M --cpu 4 --random-ports --background --prefix $provider_prefix --registry-volume $(_registry_volume) kubevirtci/${image} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
+    local params="--nodes ${KUBEVIRT_NUM_NODES} --cpu 4 --random-ports --background --prefix $provider_prefix --registry-volume $(_registry_volume) kubevirtci/${image} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
     if [[ $TARGET =~ windows.* ]]; then
-        params="--memory 8192M --nfs-data $WINDOWS_NFS_DIR $params"
+        if [[ ${KUBEVIRT_MEMORY_SIZE} == 4096 ]]; then
+            params="--memory 8192M $params"
```

Comment:
In my opinion, brute forcing difference value, when user explicitly defined some is very bad practise. If user wants to set 4GB he knows what he is doing. 

When the defaults are set as mentioned in previous comment, this if is no longer needed.